cli/vm/cli.go

```diff
@@ -297,11 +299,32 @@ Hex-encoded storage items prefix may be specified (empty by default to return th
 If seek prefix is not empty, then it's trimmed from the resulting keys.
 Items are sorted. Backwards seek direction may be specified (false by default, which means forwards storage seek direction).
 It is possible to dump only those storage items that were added or changed during current script invocation (use --diff flag for it).
+To dump the whole set of storage changes including removed items use 'dump-changes' command.
 
 Example:
 > dump-storage 0x0000000009070e030d0f0e020d0c06050e030c02 030e --backwards --diff`,
 		Action: handleDumpStorage,
 	},
+	{
+		Name: "dump-changes",
```

Comment:
`changes`? We really need to think about command naming scheme. These commands print things, but I'd like to be able to _edit_ something in the future.

Reply:
Edit storage items via CLI like it's done in debugger?

Reply:
Fixed.